packages/shell-api/src/aggregation-cursor.ts

```diff
@@ -41,6 +44,13 @@ export default class AggregationCursor extends ShellApiClass {
     return results;
   }
 
+  async [asShellResult](): Promise<ShellResult> {
+    return {
+      type: 'AggregationCursor',
+      value: this._mongo._serviceProvider.platform === ReplPlatform.JavaShell ? this : await this._asPrintable()
```

Comment:
@aherlihy this should be fine right?

Reply:
Yup, unless you want this to happen for all the classes (then we can move it into the decorators). If it works then fine by me